src/v2/view-builder/views/IdentifierView.js

```diff
@@ -74,11 +74,17 @@ const Body = BaseForm.extend({
         });
       }
 
+      // We check the 'idpDisplay' option config to determine whether to render the idp buttons 
```

Comment:
we should put this logic into it's own function now, since `render` is getting larger.. say `this.addIdpView()`
```
      // We check the 'idpDisplay' option config to determine whether to render the idp buttons 
      // above or below the login fields
      const idpDisplay = this.options.settings.get('idpDisplay');
      const isPrimaryIdpDisplay = idpDisplay && idpDisplay.toUpperCase() === 'PRIMARY';

      this.add(signInWithIdps, {
        prepend: isPrimaryIdpDisplay,
        selector: isPrimaryIdpDisplay ? '.o-form-fieldset-container' : '.o-form-button-bar',
        options: {
          idpButtons,
          isPrimaryIdpDisplay
        }
      });
    }
 ```

Reply:
yeah this is a good idea, let me make this change.